src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -49,7 +49,7 @@ public void setUp() {
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "whitespace" + File.separator + filename);
```

Comment:
Why did we remove `whitespace`? Expected directory structure should be `.../checks/whitespace/separatorwrap/...`